src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+export const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] As stated before kindly make sure `CHECK_STATUS` follow the ducks pattern.